sdk/communication/communication-tiering/test/public/getAcquiredNumberLimit.spec.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Context } from "mocha";
+import { Recorder } from "@azure-tools/test-recorder";
+import { TieringClient } from "../../src";
+import { assert } from "chai";
+import { createRecordedClient } from "./utils/recordedClient";
+
+describe(`TieringClient - Get Acquired Number Limits`, function () {
+  let recorder: Recorder;
+  let client: TieringClient;
+
+  beforeEach(async function (this: Context) {
+    ({ client, recorder } = await createRecordedClient(this));
+  });
+
+  afterEach(async function (this: Context) {
+    if (!this.currentTest?.isPending()) {
+      await recorder.stop();
+    }
+  });
+
+  it("get acquired number limits", async function () {
+    // print all acquire number limits
+    const resourceId = "9d787bd6-07fc-4c7b-8e57-17f1fee41298";
```

Comment:
I'm wondering if this would be better as an environment variable?

Also, is this supposed to be the resourceId for the ACS resource associated with the connection string? Or can I request the limits for any resource using any connection string?

Reply:
I could set it up as an environment variable.

Yea for these API's, the resourceId provided has to match the one within the connection string, if they don't match then the request throws back an error.

Reply:
> Yea for these API's, the resourceId provided has to match the one within the connection string, if they don't match then the request throws back an error. 

Probably not a topic for the SDK but for the API instead, but if the resourceId needs to match the connection string, then it might be a good idea to avoid having it as a parameter, since the backend already has access to the resourceId. That is unless this API is also intended for service to service, in which case authentication is not based around a specific resource.

Reply:
When we were developing the backend, this was also brought up, doing it this way keeps the API RESTful.

Here's a good site that goes into detail on the best practices to design an API: https://restfulapi.net/rest-api-design-tutorial-with-example/

So, we basically tried to follow this approach.